### PR TITLE
Add Emerald Ore Populator to Extreme Hills Biomes

### PR DESCRIPTION
**Pull request by GoodLucky777**
Add Emerald Ore Populator to Extreme Hills Biomes.

Difference:
  - PowerNukkit:
  Screenshot

  - Vanilla:
  Screenshot

### CODE REVIEW

File: src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public void populate(ChunkManager level, int chunkX, int chunkZ, NukkitRandom random, FullChunk chunk) {
+        int sourceX = chunkX << 4;
+        int sourceZ = chunkZ << 4;
+        int endX = sourceX + 15;
+        int endZ = sourceZ + 15;
+        
+        for (int i = 0; i < 11; i++) {
+            int x = NukkitMath.randomRange(random, sourceX, endX);
+            int z = NukkitMath.randomRange(random, sourceZ, endZ);
+            int y = NukkitMath.randomRange(random, 4, 31);
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java:27]: According to Wiki https://minecraft.gamepedia.com/Emerald_Ore#Natural_generation the Y range should be 0 - 32, is that right?

**Review by Gabriel8579 — COMMENTED** [review]
Can you confirm that the Y range is correct?

File: src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java (hunk 1 above)
Comment (GoodLucky777) [reply src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java:27]: it is weird, according to https://minecraft.gamepedia.com/Mineral_vein it was different

Comment (Gabriel8579) [reply src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java:24]: "This section is missing information about generation of emerald ore (see [here](https://gist.github.com/tryashtar/2a22bd04394275fefb6b49c348758fd6)), does this match up with the values provided on the table below?." If you enter in the url (gist) you see the math of the Y (wich is a random with max 28 + 4 which gives a range of 4 - 32)

Comment (GoodLucky777) [reply src/main/java/cn/nukkit/level/generator/populator/impl/PopulatorOreEmerald.java:24]: oh i see, i was confused

**Comment by GoodLucky777** [comment]
i fixed it

**Review by Gabriel8579 — APPROVED** [review]
Looks good now, thank you

**Pull request merged**